docs/src/index.md

```diff
@@ -129,9 +129,9 @@ julia> gradient(colordiff, RGB(1, 0, 0), RGB(0, 1, 0))
 ((r = 0.4590887719632896, g = -9.598786801605689, b = 14.181383399012862), (r = -1.7697549557037275, g = 28.88472330558805, b = -0.044793892637761346))
 ```
 
-## Gradients of ML models
+## Explicit and Implicit Parameters
 
-It's easy to work with even very large and complex models, and there are few ways to do this. Autograd-style models pass around a collection of weights.
+It's easy to work with even very large and complex models, and there are few ways to do this. Autograd-style models pass around a collection of weights. There are two ways of passing *explicit* parameters:
```

Comment:
```suggestion
It's easy to work with even very large and complex models, and there are few ways to do this. Autograd-style models pass around a collection of weights. Depending on how you write your model, there are multiple ways to *explicity* take gradients with respect to parameters. For example, the function `linear` accepts the parameters as an argument to the model. So, we directly pass in the parameters, `θ`, as an argument to the function being differentiated.
```

Reply:
For some reason, I cannot comment on lines that aren't changed by the PR (which is silly). But here is my suggestion for L156 that pairs with this one.

```
An extension of this is the Flux-style model in which we use call overloading to combine the weight object with the pullback pass (equivalent to a closure). Here, we take the gradient with respect to `model` which is an instance of the `Linear` type. Zygote returns a `NamedTuple` of gradients which shares the same structure as `Linear` (the keys of the tuple correspond to the fields of the struct).